packages/shared/src/enterprise/licenseUtil.ts

```diff
@@ -147,6 +158,19 @@ export interface LicenseInterface {
     discountMessage?: string; // The message of the discount
     hasPaymentMethod?: boolean;
   };
+  orbSubscription?: {
+    id: string;
+    customerId: string;
+    qty: number;
+    trialEnd: Date | null;
+    status: Stripe.Subscription.Status;
+    current_period_end: number;
+    cancel_at: number | null;
+    canceled_at: number | null;
```

Comment:
I'm not sure this is going to be possible to extract from Orb as it isn't stored on their `subscription` object. They do have an endpoint to fetch subscription schedules, but without testing, I'm not confident that would also have the information we want.

Reply:
So we can store the `canceled_at` timestamp, but it might get cumbersome to manage the state.

We'll need to update `updateOrbSubscriptionInDb` to persist the state unless an org re-subscribes at a certain point. Not impossible, but we'll have to manage it manually (as opposed to stripeSubscriptions) where they manage it.